addons/analytic/models/analytic_plan.py

```diff
@@ -46,6 +47,7 @@ def _default_color(self):
         'res.company',
         string='Company',
         default=lambda self: self.env.company,
+        domain=lambda self: [('id', 'in', self.env.context.get('allowed_company_ids'))]
```

Comment:
Ẁhy?

Reply:
Hello @william-andre  Thank you for looking into it. :slightly_smiling_face: 

If we select a company outside of our current environment, an access error occurs.
LInk: https://i.imgur.com/1znLDKC.png

Reply:
We have the same behavior on `res.partner` for instance; I'm not sure we want to fix it this way...
An idea fix would be to allow it, but to return to the list view for instance. (I'm not asking you to do it, it should be a "big" task on it's own)